apps/src/app.c

```diff
@@ -877,6 +957,7 @@ int32_t run_app(FlowInfo flow_infos[], uint32_t num_flows, CmdArgs *cmd_args)
                 {
```

Comment:
This should be if instead of else if
There will be cases where there are 2 buffers dequeued
You need to enqueue back the one dq_bufs as well

Reply:
Done